addons/missileguidance/XEH_postInit.sqf

```diff
@@ -3,41 +3,43 @@
 
 [QGVAR(handoff), LINKFUNC(handleHandoff)] call CBA_fnc_addEventHandler;
 
+if (!hasInterface) exitWith {};
+
 ["ACE3 Weapons", QGVAR(cycleFireMode), LLSTRING(CycleFireMode), {
```

Comment:
should this be a keybind?

Reply:
I think it should already handle player and vehicle weapons, as long as they have different fire-modes / `useModeForAttackProfile`
but we can always move it over if we find a need